src/amber/cli/generators/scaffold.cr

```diff
@@ -7,17 +7,84 @@ module Amber::CLI
     property view : Generator
 
     def initialize(name, fields)
-      super(name, fields)
-      if config.model == "crecto"
-        @migration = CrectoMigration.new(name, fields)
-        @model = CrectoModel.new(name, fields)
-        @controller = ScaffoldCrectoController.new(name, fields)
+      # Keywords from https://github.com/crystal-lang/crystal/wiki/Crystal-for-Rubyists#available-keywords as of May 6th, 2019
+      keywords = [
+        "abstract",
+        "do",
+        "if",
+        "nil?",
+        "select",
+        "union",
+        "alias",
```

Comment:
This list seems to be of method names even though constants being defined. It would technically be sort of ok to have a model called "Unless" other than what it would do to possible associations.

Reply:
In the chat we discussed (https://gitter.im/amberframework/amber?at=5cd0b8f5375bac7470697ff2) potentially moving this logic over into `src/amber/cli/generators.cr` since some validations for the name already occur there. Would you be opposed to moving this logic into that file and outputting a warning (e.g. "It looks like you may be using a keyword.") but still allowing it to complete?

Reply:
No that sounds good as long as it doesn't increase code complexity substantially.

Reply:
Okay I've made changes that should address your concerns. Let me know if there's any other changes you believe are necessary. With the only logical change being a puts statement to output a warning do we need to add additional specs for this change?